CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing to CICE
+
+Welcome to the CICE Consortium ! First, thanks a lot for considering to contribute to CICE.
```

Comment:
Nit picky thing, sorry. Change to:

Welcome to the Consortium! Thank you for your interest in contributing to CICE.

Reply:
done